src/Common/Survey/ModuleInfo.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.WindowsAzure.Commands.Common.Survey
+{
+    public class ModuleInfo
+    {
+        [JsonProperty(PropertyName = "name")]
+        internal string Name { get; set; }
+
+        [JsonProperty(PropertyName = "version")]
+        internal int Version { get; set; }
+
+        [JsonProperty(PropertyName = "count")]
+        internal int Count { get; set; }
```

Comment:
Could use some explaination
```suggestion
        /// <summary>Count of ???</summary>
        [JsonProperty(PropertyName = "count")]
        internal int Count { get; set; }
```